### PR TITLE
feat: add pagination

### PR DESCRIPTION
**Pull request by tenadopt**
Add pagination for 15 images.

### CODE REVIEW

File: src/entities/image/VirtualizedImageList.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+                        <IconButton
+                            aria-label="delete"
+                            onClick={(e) => {
+                                e.stopPropagation();
+                                onDelete(image.id);
+                            }}
+                        >
+                            <DeleteIcon />
+                        </IconButton>
+                    </CardContent>
```
Comment (shev7) [inline src/entities/image/VirtualizedImageList.tsx]: move image to separate component, if its related to work it shold be `WorkCard` or smth like this

File: src/entities/image/VirtualizedImageList.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+import DeleteIcon from '@mui/icons-material/Delete';
+import { Image } from "features/imageGallery/api/api";
+
+interface ImageGridProps {
+    images: Image[];
+    onEdit: (id: string) => void;
+    onDelete: (id: string) => void;
+}
+
+const ImageGrid = ({ images, onEdit, onDelete }: ImageGridProps) => (
```
Comment (shev7) [inline src/entities/image/VirtualizedImageList.tsx]: shouldn't this be `WorkGrid`

File: src/pages/ImageFeed/ui/ImageFeedPage.tsx [hunk 3]
```diff
@@ -1,28 +1,30 @@
-import React from 'react';
+import React, {useState} from 'react';
```
Comment (shev7) [inline src/pages/ImageFeed/ui/ImageFeedPage.tsx]: not formatted

File: src/entities/image/VirtualizedImageList.tsx
Comment (shev7) [inline src/entities/image/VirtualizedImageList.tsx:1]: Shouldn't this be in`ui` segment of entity, and entity should be named like work i guess, its not an image, it is a work

Comment (shev7) [reply src/entities/image/VirtualizedImageList.tsx:1]: please follow fsd architecture

**Review by shev7 — CHANGES_REQUESTED** [review]
pagination should look like this 
image
And current page must be used from search params, so you don't need local component state, just use search params

File: src/entities/image/VirtualizedImageList.tsx (hunk 1 above)
Comment (tenadopt) [reply src/entities/image/VirtualizedImageList.tsx]: I rename to Image Card. Nodody makes mistake.

File: src/entities/image/VirtualizedImageList.tsx
Comment (tenadopt) [reply src/entities/image/VirtualizedImageList.tsx:1]: I made the image as entity

File: src/entities/image/VirtualizedImageList.tsx (hunk 2 above)
Comment (tenadopt) [reply src/entities/image/VirtualizedImageList.tsx]: I make the image as entity.

**Review by shev7 — APPROVED** [review]

**Pull request merged**